packages/server/test/integration/video_capture_spec.ts

```diff
@@ -0,0 +1,41 @@
+const { expect, sinon } = require('../spec_helper')
+import videoCapture from '../../lib/video_capture'
+import path from 'path'
+import fse from 'fs-extra'
```

Comment:
there's a `packages/fs` you could use... but i will yell at you for using `*sync` methods